packages/math-input/src/components/keypad/keypad.tsx

```diff
@@ -0,0 +1,171 @@
+import Color from "@khanacademy/wonder-blocks-color";
```

Comment:
I don't think anything really changed here. I just wanted to keep `index.ts` focused on exporting the pieces.

Reply:
In the future, doing this in two PRs would make this easier to review (file moves and then actual code changes).  Your comment helps though! Thanks. :)